index.html

```diff
@@ -198,15 +198,24 @@ <h2>Contact me</h2>
             coding.
             I’d love to help with it!
         </p>
-        <form id="contac-form" class="contac-form" action="">
-            <label for="lorem"><input type="text" name="lorem ipsum" id="lorem" placeholder="Lorem Ipsum"></label>
-            <label for="email"><input type="email" name="email" id="email-input"
-                    placeholder="loremipsum@gmail.com"></label>
-            <label for="user-message">
-                <textarea name="user-message" id="user-message" cols="10" rows="5"
-                    placeholder="Write your message here..."></textarea>
-            </label>
-            <input type="submit" name="get-in-touch" id="get-in-touch" value="Get in touch">
+        <form id="contac-form" class="contac-form" action="https://formspree.io/f/mbjewygq" method="POST">
+            <ul>
+                <li>
+                    <label for="name"><input type="text" name="name" id="name" placeholder="name" maxlength="30" required></label>
+                </li>
+                <li>
+                    <label for="email"><input type="email" name="email" id="email-input"
+                    placeholder="email@domain.com" required></label>
+                </li>
+                <li>
+                    <label for="user-message">
+                        <textarea name="user-message" id="user-message" cols="10" rows="5"
+                        placeholder="Write your message here..." maxlength="500" required></textarea>
+                    </label>
+                </li>
+                <!-- <input type="submit" name="get-in-touch" id="get-in-touch" value="Get in touch"> -->
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)